scrapy/extensions/feedexport.py

```diff
@@ -151,7 +151,7 @@ def __init__(self, settings):
         if not self._exporter_supported(self.format):
             raise NotConfigured
         self.store_empty = settings.getbool('FEED_STORE_EMPTY')
-        self.export_fields = settings.getlist('FEED_EXPORT_FIELDS')
+        self.export_fields = settings.getlist('FEED_EXPORT_FIELDS') or None
```

Comment:
`settings.getlist('FEED_EXPORT_FIELDS', None)` cover the case of the user using an empty list.